browser/Makefile.am

```diff
@@ -152,6 +153,13 @@ NODE_MODULES_SRC =\
 	l10n-for-node@0.0.1 \
 	@braintree/sanitize-url@6.0.0
 
+COOL_SETTING_IFRAME_TS = \
+	setting_iframe/src/WordBook.ts \
+	setting_iframe/src/Xcu.ts \
+	setting_iframe/src/AdminIntegratorSettings.ts
```

Comment:
Followup - rename `AdminIntegratorSettings.ts` as it's used for user and admin setting both.